pkg/state/score.go

```diff
@@ -0,0 +1,66 @@
+package state
+
+import (
+	"math/big"
+
+	"github.com/pkg/errors"
+	"github.com/wavesplatform/gowaves/pkg/keyvalue"
+)
+
+func calculateScore(baseTarget uint64) (*big.Int, error) {
+	if baseTarget == 0 {
+		return nil, errors.New("zero base target")
+	}
+	var max big.Int
+	max.SetString("18446744073709551616", 10)
```

Comment:
Maybe extract `max` to global var? Is it possible initialise it not from string?

Reply:
It's only possible to initialize it from string or from bytes slice, since it's bigger than `MaxUint64`.